index.html

```diff
@@ -43,6 +43,122 @@ <h4>LET'S CONNECT</h4>
+   <div class="card three">
+    <div class="placeholder-img3"></div>
+    <div class="text">
+        <h3>Tonic</h3>
+        <ul class="tech-dev">
+            <li><a href="#">CANOPY</a></li>
+            <li><a href="#">Back End Dev</a></li>
+            <li><a href="#">2015</a></li>
+        </ul>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="lang-names">
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+        </ul>
+        <button type="button">See Project</button>
+    </div>
+   </div>
+   </div>
+   <div class="container4">
+   <div class="card four">
+    <div class="placeholder-img4"></div>
+    <div class="text">
+        <h3>Multi-Post Stories</h3>
+        <ul class="tech-dev">
+             <li><a href="#">CANOPY</a></li>
+            <li><a href="#">Back End Dev</a></li>
+            <li><a href="#">2015</a></li>
+        </ul>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="lang-names">
+             <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+        </ul>
+        <button type="button">See Project</button>
+    </div>
+   </div>
+   </div>
+   </section>
+   <section class="about">
+    <div class="abt-myself">
+    <div class="about-sec">
+        <h1>About Myself</h1>
+        <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+    </div>
+    <div class="social-media-btn">
+        <h4>LET'S CONNECT</h4>
+        <div class="icons">
+            <ul>
+                <li><a href="https://github.com/fauziyahzeeyah"><i class="fa-brands fa-github"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-linkedin"></i></a></li>
+                <li><a href="https://medium.com/"><i class="fa-solid fa-m"></i></a></li>
+                <li><a href="https://twitter.com/FauziyahZeeyah?t=5x4tHwdJG62bloZ3QehDcw&s=08"><i class="fa-brands fa-twitter"></i></a></li>
+                <li><a href="https://angel.co/"><i class="fa-brands fa-angellist"></i></a></li>
+            </ul>
+        </div>
+        <button type="button"><a href="#">Get My Resume</a></button>
+    </div>
+    </div>
+   <div class="dropdowns">
+    <ul class="dropdown-menus">
+        <li class="langs">Languages <img src="images/arrow down.svg" alt="" class="down-arr"></li>
+        <ul class="drop-downs">
+                <li><img src="./images/js.png" alt=""><span>Javascript</span></li>
+                <li><img src="./images/html.png" alt=""><span>HTML</span></li>
+                <li><img src="./images/css.png" alt=""><span>CSS</span></li>
+            </ul>
```

Comment:
- [ OPTIONAL ] As I can see here there's a space between the icon and the HTML so kindly add more space to match the Figma design

- Your design
![image](https://user-images.githubusercontent.com/58553711/200141014-b1f0e5d7-3588-44b4-a00a-477ec4cefcb8.png)

- Figma
![image](https://user-images.githubusercontent.com/58553711/200141020-84e322f7-6228-4e8c-b374-7634d9b84861.png)